### ISSUE TITLE
t/130-search.t fails 'usageper' test if all disks are less than 0.5% full

### ISSUE THREAD

**Issue by jkahrman** [glasswalk3r/Linux-Info#5]
Filesys::Df::df can return 0 in the 'per' field if the 'used' field is less than 0.5% of the 'total' field. This results in t/130-search.t failing if all disks are less than 0.5% full

% df /                                                                       Filesystem     1K-blocks    Used Available Use% Mounted on
/dev/sdb2      667606776 1809464 631815056   1% /

% perl -d
x Linux::Info::DiskUsage->new->get
0  HASH(0xaaab0e793000)
   '/dev/sdb2' => HASH(0xaaab0e9833b8)
      'free' => 665797312
      'mountpoint' => '/'
      'total' => 633624520
      'usage' => 1809464
      'usageper' => '-'

t/130-search.t
1..11
ok 1 - checking cpustats
ok 2 - checking procstats
ok 3 - checking memstats
not ok 4 - disk usage info is available
\#   Failed test 'disk usage info is available'
\#   at ./t/130-search.t line 34.
not ok 5
\#   Failed test at ./t/130-search.t line 35.
\#          got: ''
\#     expected: 'HASH'
ok 6 - checking system
ok 7 - checking total
ok 8 - checking idle
ok 9 - checking nice
ok 10 - checking user
ok 11 - checking iowait
\# Looks like you failed 2 tests of 11.

**Comment by glasswalk3r** [comment]
I believe that `Filesys::Df::df` is failing to comply with it's defined interface (which is, of course, too relaxed).
I don't think that this kind of error should be handled by `Linux::Info`.
Did you try to reach out for the people responsible for `Filesys::Df`? It seems the project is not being updated anymore though.

**Comment by glasswalk3r** [comment]
Indeed, that are very old bugs opened without response: https://rt.cpan.org/Public/Dist/Display.html?Name=Filesys-Df.
Again, please try to reach out for https://metacpan.org/author/IGUTHRIE and see if he is willing to help or even transfer ownership of the `Filesys::Df` namespace on Metacpan, so we can try to fix this over there.
